crates/turborepo-lib/src/commands/link.rs

```diff
@@ -394,8 +394,9 @@ fn enable_caching(url: &str) -> Result<()> {
     Err(anyhow!("link after enabling caching"))
 }
 
-fn add_turbo_to_gitignore(base: &CommandBase) -> Result<()> {
-    let gitignore_path = base.repo_root.join_component(".gitignore");
+fn add_turbo_to_gitignore(repo_root: &AbsoluteSystemPathBuf) -> Result<()> {
+    let turbo_test_regex = Regex::new("^.turbo/? *$").unwrap();
```

Comment:
- No user input, `unwrap` is safe.
- Uses the precise definition of whether or not `.turbo` is ignored.

Reply:
Can we use expect instead? even just 'known regex' is probably enough.